chapters/ch41.md

```diff
@@ -1,99 +1,99 @@
 > # 41. Identifying Bias, Variance, and Data Mismatch Errors
 
---> _replace THIS LINE by your translation for the above line_
+# 41. Xác định lỗi độ chệch, phương sai và dữ liệu không khớp
 
 
 > Suppose humans achieve almost perfect performance (≈0% error) on the cat detection task, and thus the optimal error rate is about 0%. Suppose you have:
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử con người đạt được chất lượng gần như hoàn hảo (lỗi ≈0%) trong việc phát hiện mèo, và do vậy tỷ lệ lỗi tối ưu là khoảng 0%. Giả sử bạn có:
 
 > * 1% error on the training set.
 
---> _replace THIS LINE by your translation for the above line_
+* 1% lỗi trên tập huấn luyện.
 
 
 > * 5% error on training dev set.
 
---> _replace THIS LINE by your translation for the above line_
+* 5% lỗi trên tập phát triển huấn luyện.
 
 
 > * 5% error on the dev set.
 
---> _replace THIS LINE by your translation for the above line_
+* 5% lỗi trên tập phát triển.
 
 > What does this tell you? Here, you know that you have high variance. The variance reduction techniques described earlier should allow you to make progress.
 
---> _replace THIS LINE by your translation for the above line_
+Như vậy nói lên được điều gì? Ở đây, bạn biết rằng bạn đang có phương sai cao. Các kỹ thuật giảm phương sai được mô tả trước đây sẽ cho phép bạn tiến bộ lên.
 
 
 > Now, suppose your algorithm achieves:
 
---> _replace THIS LINE by your translation for the above line_
+Bây giờ, giả sử thuật toán của bạn đạt được:
 
 
 > * 10% error on the training set.
 
---> _replace THIS LINE by your translation for the above line_
+* 10% lỗi trên tập huấn luyện.
 
 
 > * 11% error on training dev set.
 
---> _replace THIS LINE by your translation for the above line_
+* 11% lỗi trên tập phát triển huấn luyện.
 
 
 > * 12% error on the dev set.
 
---> _replace THIS LINE by your translation for the above line_
+* 12% lỗi trên tập phát triển.
 
 
 > This tells you that you have high avoidable bias on the training set. I.e., the algorithm is doing poorly on the training set. Bias reduction techniques should help.
 
---> _replace THIS LINE by your translation for the above line_
+Điều này cho bạn biết rằng bạn có độ chệch trên tập huấn luyện cao mà bạn có thể tránh được. Tức là, thuật toán đang hoạt động kém trên tập huấn luyện. Kỹ thuật giảm độ chệch sẽ có ích trong trường hợp này.
 
 
 > In the two examples above, the algorithm suffered from only high avoidable bias or high variance. It is possible for an algorithm to suffer from any subset of high avoidable bias, high variance, and data mismatch. For example:
 
---> _replace THIS LINE by your translation for the above line_
+Trong hai ví dụ trên, thuật toán chỉ có vấn đề về độ chệch cao nhưng có thể tránh được hoặc về phương sai cao. Một thuật toán có thể mắc phải một hoặc nhiều vấn đề về độ chệch cao có thể tránh được, phương sai cao hoặc dữ liệu không khớp. Ví dụ như:
```

Comment:
*Mình thấy có thể thêm dấu phẩy để ngắt cho rõ ý hơn:
- ".... thuật toán chỉ có vấn đề về độ chệch cao nhưng có thể tránh được, hoặc về phương sai cao."